packages/tasks/src/model-libraries.ts

```diff
@@ -223,6 +223,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			terms: { path: ["ckpt/tensor00000_000", "ckpt-0/tensor00000_000"] },
 		},
 	},
+	"hunyuan-dit": {
+		prettyLabel: "HunyuanDiT",
+		repoName: "HunyuanDiT",
+		repoUrl: "https://github.com/Tencent/HunyuanDiT",
+		countDownloads: {
+			terms: { path: ["pytorch_model_ema.pt", "pytorch_model_distill.pt"] },
```

Comment:
Is there any file that is consistently across repos? E.g. vocab.txt or a config.json?

Reply:
Not really. They chose not to add a metadata file, only using weight file for download metrics. They suggested to use the two .pt files are the two main weight file, one for normal weight and another one for distilled weight. The two files are mutually exclusive and won't be seen in the same repo.